package-dev/Runtime/SentryInitialization.cs

```diff
@@ -13,7 +13,7 @@
 using UnityEngine;
 using UnityEngine.Scripting;
 
-#if SENTRY_NATIVE_IOS
+#if SENTRY_NATIVE_COCOA
 using Sentry.Unity.iOS;
```

Comment:
Should we change the namespace?

Reply:
Yes, but didn't want to do that in the same PR to avoid polluting the changelist, if it didn't show up just as "renames". I'll do that in a followup PR